### PR TITLE
Contact form page bug bash

### LINKED ISSUE #49: Contact us page bug bash
reexport the piggy img with a better resolution

### PR DESCRIPTION
**Pull request by Luey-A**
close #49 
- replaced the img at the bottom with another one with a higher resolution.
- made the radius for all the inputs, imgs, form button and form as 0.5em

Screen Shot 2020-11-22 at 17 13 19
Screen Shot 2020-11-22 at 17 33 14
Screen Shot 2020-11-22 at 17 34 37

### CODE REVIEW

File: src/components/ContactForm/index.jsx [hunk 1]
```diff
@@ -114,7 +114,11 @@ export default function ContactForm() {
         </Col>
       </Row>
       <Row className="contactPageImgsRow">
-        <img className="contactPageImgs" src={PiggyImg} alt="Piggy pig" />
+        <img
+          className="contactPageImgs"
+          src={WomenCheckingPhoneImg}
+          alt="Man with headphones"
```
Comment (Ammar-64) [inline src/components/ContactForm/index.jsx]: Why the alt is different from src

**Review by Ammar-64 — CHANGES_REQUESTED** [review]
Can you please fix the alt

File: src/components/ContactForm/index.jsx (hunk 1 above)
Comment (Luey-A) [reply src/components/ContactForm/index.jsx]: fixed. PTAL

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Review by olaouso — APPROVED** [review]

**Pull request merged**